src/plugins/dev_tools/public/types.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export interface ConfigSchema {
+  serverless: {
```

Comment:
Please don’t do this.

We’re avoiding “am I serverless?” checks and schema in order to divide our functional decisions based on serverless.

So the question and config should be, “should we be using deep links”, and perhaps `config.deeplinks.visible`.

Reply:
Yeah sounds good, I've addressed your feedback with https://github.com/elastic/kibana/pull/161227/commits/b419c0506f57c1f2cfb60c56b51fce5c24a55b9c let me know what you think!